Civil-Service-Human-Resources/rpg-careers-wp
# 396 add theme

__What__

This PR merges the Theme repo into this repo, retaining the commit history and moving the theme files into a theme folder.

__How to test__

1. Review the git log ensure that commits are shared with the theme repo
2. Ensure that all theme files are included.

__Who can review__
@ianrholmes
